e2e/specs/program-enrollment.spec.ts

```diff
@@ -75,8 +75,13 @@ test('Add and edit a program enrollment', async ({ page }) => {
   });
 
   await test.step('When I click on the `Edit` button of the created program', async () => {
-    await programsPage.overflowButton().click();
-    await programsPage.editProgramButton().click();
+    await expect(dataRow).toBeVisible();
```

Comment:
We already have a bunch of assertions about the `dataRow` in the previous step. Why do we need one here?